### ISSUE TITLE
lifestyle ponderings

### ISSUE THREAD

**Issue by serapath** [nicksellen/ponderings#7]
digital nomad is the way to go for everyone.
- kills country borders
- makes the world one
- lets us look to the sky and explore the outside
- makes everyone have a geopolitical mindset and much more pro-active towards inequality

People do not need to fly or drive, they can walk or cycle. If i can go everywhere and are connected to the internet all the time, all the places in between two locations are interesting.

But the thing is, until the "money issue" is not solved, people will not ever be able to become digital nomads.

**Comment by nicksellen** [comment]
What about people that don't/can't do digital work?

**Comment by serapath** [comment]
like which? :-) There are currently probably some people for which that is true, but its all going to be digital. ... or lets say digital work will become more normal again.
Saw a nice video on "Wizard Amigos" :-) https://www.youtube.com/watch?v=w2itwFJCgFQ

**Comment by serapath** [comment]
Those people who cannot work digitally enhanced yet, can work on that and every little success will them make a little bit more location independent :-)

**Comment by nicksellen** [comment]
Nice quadcoptor video - I added it to https://github.com/nicksellen/ponderings/blob/master/future.md

On jobs that aren't digital, here's a few: Actor, Animal Keeper, Announcer, Archeologist, Artist, Athlete, Car Mechanic, Carpenter, Chef, Chemist, Childcare worker, Dancer, Dentist, Diplomat, Doctor, Electrical Engineer, Farmer, Fashion Designer, Firefighter, Fitness Trainer, Gardener, Judge, Marine Biologist, Musician, Photographer, Police Offier, Pilot, Psychologist, Botanist, Reporter, Therapist, Teacher, Veterinarian, Writer.

> its all going to be digital

I think computers are amazing and we can do wonderful things with them, and there is a lot more to come (I added some cool/scary robotics links on the [future](https://github.com/nicksellen/ponderings/blob/master/future.md) page), but that in itself doesn't suggest all jobs will be digital. It could actually go the opposite way, where computers do all their own "digital work", and leave us to do things they really can't do (Therapist?).

> makes the world one

I would love for the world to be one. What I would worry about if everybody were to become kind of transient is how we would form real long lasting communities.

> digital nomad is the way to go for everyone

Some people probably don't actually _want_ to be a digital nomad. Sometimes for just simple practical reasons, health or disability. And not everybody wants to sit at a computer all the time.

I like the world where people do a million and one different things.

**Comment by nicksellen** [comment]
@serapath I saw you linked to http://allcodesarebeautiful.com/en/digital-nomadism-vision/ on facebook, which isn't as straightward about the benefits of digital nomadism as you expressed, what did you think of the article?

**Comment by serapath** [comment]
I dont like self centered digital nomads a lot, who just travel like permanent tourists, doing remote work on the side. The point would be to help empower those who cannot follow. I think a digital nomad should be "on a mission" :-) But it's also difficult, because afaik the laws in countries don't make it easy to work somewhere on a "local business".  The article has lots of people with different perspectives on it.

I think the criticism is not about digital nomadism, but about the world we live in. Digital nomads can work on making things better for everyone, because they have the chance to actually become part of the life of people who are not priviliged and help them make a difference

**Comment by nicksellen** [comment]
Given the existing digital nomads don't actually manage the positive points in your opening comment, can you justify the opening statement?

> digital nomad is the way to go for everyone

**Comment by serapath** [comment]
Some do, some dont and just by going to those countries, they buy something and inspire locals and eventually start to care, while people not even going there might not care about any conditions.

But anyway. Digital nomads or perpetual travelers - if everyone was one, would force countries to work together, because people would not think national any longer and would make the world come closer together and if its only for fixing tax laws.

**Comment by nicksellen** [comment]
_Digital_ nomad != perpetual traveler

The _digital_ bit is the key thing that enables digital nomads to be wanderers - they can keep a reasonable income by having clients/customers in first-world countries. Not everybody can do that - like all those jobs I listed above.

**Comment by serapath** [comment]
I think a "remote job" is what makes people mobile.
Iv most people had "remote jobs", they could move freely and that would change how country borders are perceived. I think people would feel less bound to certain locations and feel more connected to people from all over the world. I guess thats a good thing.
How to call it, i don't care. If there are ways to work "remotely" without being "digital", then that's fine with me. In the past, that obviously didn't work for the mainstream.

You might want to check some statistics, but "knowledge workers" make a major growing trend in the developed world and maybe in the future this trend will continue and eventually most people will be "knowledge workers" using a computer (whatever computers of the future would look like and how they would be used).

Apart from that, i suggest to define some kind of questions that guide the discussions, because i often get the feeling they drift randomly and I wonder if its discussion just for the sake of discussion.

**Comment by nicksellen** [comment]
I agree there is a shift to a more flexible working location for some people, and this will probably increase in time. I think we agree about this. A future where _everybody_ works solely on the computer is so far away to be difficult to predict with any accuracy, and also not a world I want to be part of.

_Knowledge worker_ doesn't imply location independent either. From the [wikipedia](http://en.wikipedia.org/wiki/Knowledge_worker) definition it includes doctors, and these are very much not location independent, they need to be next to the patient they are treating! Sure, there are patients around the world they could attend to, but the doctor would be very much dependent on their current location.

**Comment by nicksellen** [comment]
Discussion for the sake of discussion is fine for me! Life is only for sake of life anyway... I would like to get a more structured approach to philosophising/pondering (was brainstorming a bit today about it actually), but discussion for the sake of discussion will probably always be involved, it's one of lifes great pleasures surely?

**Comment by serapath** [comment]
Why wouldnt you want to be part of such a world?
I'm never talking about a world were people would not do all kinds of activity for fun, I'm ALWAYS just thinking of professional activities, activities to supply the economy with all kinds of goods and services as efficient as possible, so that people have as much time as possible on their hands to spend them in creative ways of their choosing.

Regarding the example of a doctor, I would hope there will be always equipment and doctors around or I could find the nearest doctor with appropriate equipment through an app. But regarding my medical records, i'd like to keep them online and get them analysed and send through all the specialists that are needed and they basically its processed by computers where the algorithms are constantly updated according to current state of the art research and if any anomalies show up, specialised doctors "debug" them.

**Comment by serapath** [comment]
My primary goals when it comes to discussing philosophy is application, where my main driver is to figure out how to design a world in which it's possible to automate the economy in a way, so that everybody can engage in learning and creative work all the time and once solutions have been found they get automated and taken care of by the automated economy.

Basically to bring everyone to the level of "75k USD/GBP/EUR" or whatever the magical number is that makes it possible to only engage in learning tasks and creatives tasks and not competitive and repeating tasks.

**Comment by nicksellen** [comment]
> Why wouldnt you want to be part of such a world?

Working with your hands is satisfying and you can't always replace it with the _fun_ version of the activity, because it has no purpose/utility that way.

In the same way I like cycling to/from work as it serves a purpose, if I work from home I stop cycling. Doing a little loop in the morning starting and ending at home doesn't do it for me, it has no purpose.

> Regarding the example of a doctor, I would hope there will be always equipment and doctors around or I could find the nearest doctor with appropriate equipment through an app.

Is the vision that there are equally well trained doctors covering the planet, working within a worldwide healthcare system, and they all speak English/German?, and have worldwild secure/trusted access to your patient records, and there is mobile data coverage too.

This would be a _huge_ leap from our current world.

It's _easy_ to have a technological vision of the future where computers do everything, people have been doing that for decades. And we are living in somebodies dream right now - we actually _already have_ the capabilities for everybody on the planet to have a reasonable standard of living, but we aren't putting them to this use.

We perhaps need technological progress, but more importantly I think we need political progress. There are big power structures in the world (governments, business, etc) that are quite happy with the current world setup, and will be quite happy to absorb more technology and use it to further their goals.

**Comment by serapath** [comment]
#### purpose

It has purpose/utility that way. You build a house from wood with your friends and camp in it over the weekend. That has nothing to do with the fact, that you could do the same things faster, better and more efficient if professional people do the same thing with proper equipment. And only because those people could do it, it wont take any fun away from you.

#### cycling

Cycling to and from home is maybe efficient, because of traffic lights its faster, and its "multi purpose", so you get some fresh air and sun and you do some sporty activity + you arrive at work, so if you would not do all of that combined, you might need to spend some extra time later to make up for it.

#### doctors

there are extremely specialized doctors, working with the internet (you call it "worldwide healthcare system") and they speak all english (because if they studied, they know english already, but otherwise translation services will improve) .. i doubt they will speak german (only to some end customers which dont speak english or at least not well, there some translation into local native language will happen)

Regarding access to patient records - i wish for having control over my record and i would share parts of it as i see it fit with the doctors i want to share it with. It will probably be an "opt-in" service, where those who want to use it can share their medical records and those who dont just simply dont :-)

#### power structures :-)

i dont know much about it - there are many rumors that tell all kinds of stories.
I do not think the current political system will enable any kind of change. Its either all those who sit in the same boat without access to ressources, who build an alternative in an opt-in entrepreneurial way, utilizing maybe blockchain, p2p and other technologies as they seem fit... or it simply wont happen.
